PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -8,6 +8,28 @@
 
 from PhysicsTools.Heppy.physicsutils.QGLikelihoodCalculator import QGLikelihoodCalculator
 
+def findBin(v, vals):
+    find=False;
+    bm=0;
+    bM=len(vals)-1;
+    if v<vals[bm]: return -1
+    if v>=vals[bM]: return len(vals)
+    n=0;
+    while not find :
+      if v<vals[bm+(bM-bm)/2]:
+          bM=bm+(bM-bm)/2
+      else:
+          bm=bm+(bM-bm)/2
+      if abs(bM-bm)==1:
+	return bm
+         
+      if n>len(vals):
+          return -1
+      n+=1
+    
+    return -1;
+
+
```

Comment:
all this is not needed, just add "from bisect import bisect"
https://docs.python.org/2.7/library/bisect.html